src/views/v2/components/messages.php

```diff
@@ -0,0 +1,31 @@
+<?php
+/**
+ * View: Messages
+ *
+ * Override this template in your own theme by creating a file at:
+ * [your-theme]/tribe/events/views/v2/components/messaging.php
+ *
+ * See more documentation about our views templating system.
+ *
+ * @link {INSERT_ARTCILE_LINK_HERE}
+ *
+ * @version TBD
+ *
+ * @var array $messges An array of user-facing messages, managed by the View.
+ */
+
+if ( empty( $messages ) ) {
+	return;
+}
+
+// @todo @juanfra Make this beautiful.
+?>
+
+<div class="tribe-events-notices" style="text-align: center; width: 100%; margin: 0 0 1em auto;">
+	<?php foreach ( $messages as $message_type => $message_group ) {
+		foreach ( $message_group as $message ) {
+			echo "{$message_type}: $message";
+		}
```

Comment:
* :warning: Expected next thing to be an escaping function (see Codex for 'Data Validation'), not '"{$message_type}: $message"'
* :no_entry_sign: All output should be run through an escaping function (see the Security sections in the WordPress Developer Handbooks), found '"{$message_type}: $message"'.

<sub>hash: 4c20c7691851f593f0e710ab8e530422</sub>